cfme/fixtures/automate.py

```diff
@@ -63,3 +66,21 @@ def method(ruby_code):
         return instance
 
     return method
+
+
+@pytest.fixture()
+def import_datastore(appliance, request):
+    """This fixture will help to import datastore file"""
+
+    def domain(file_name, from_domain, to_domain=None):
+        # Download datastore file from FTP server
+        fs = FTPClientWrapper(cfme_data.ftpserver.entities.datastores)
+        file_path = fs.download(file_name)
+
+        # Import datastore file to appliance
+        datastore = appliance.collections.automate_import_exports.instantiate(
+            import_type="file", file_path=file_path
+        )
+        request.addfinalizer(datastore.delete_if_exists)
```

Comment:
Hey @digitronik, `delete_if_exists` is failing here because `domain.exists` returns False. I think you will have to either override `AutomateFileImport`'s `exists` method or define a navigator to `Details` page of the datastore to make this work.

Reply:
@valaparthvi no, problem not with `AutomateFileImport`; its not entity having details page :). but we need to delete `domain` which create by import means tear-down should act on what changes create by test.

Reply:
But don't you think deleting that imported datastore is necessary? Currently, it just stays as is. Have you tried deleting the datastore via REST? If the domain is not locked, you can delete it via REST.

Reply:
Ay, got it! Tests are working fine now. Thanks for clearing it up :)